### ISSUE TITLE
Close oustanding pending ISP reports

### ISSUE THREAD

**Issue by Mike-ORG** [openrightsgroup/Blocking-Middleware#150]
The following ISP reports can't be closed out because they are pending:

https://www.blocked.org.uk/control/ispreports/Vodafone/http://www.perkinsdrugs.com
https://www.blocked.org.uk/control/ispreports/BT-Strict/http://www.ozee.com
https://www.blocked.org.uk/control/ispreports/BT-Strict/http://www.instagram.com
https://www.blocked.org.uk/control/ispreports/O2/http://web.archive.org
https://www.blocked.org.uk/control/ispreports/BT-Strict/http://www.linkedin.com
https://www.blocked.org.uk/control/ispreports/O2/http://www.debunker.com
https://www.blocked.org.uk/control/ispreports/BT/http://www.middletonbell.co.uk
https://www.blocked.org.uk/control/ispreports/EE/http://www.cbdoilsuk.com
https://www.blocked.org.uk/control/ispreports/Three/http://www.cbdoilsuk.com
https://www.blocked.org.uk/control/ispreports/Vodafone/http://www.cbdoilsuk.com
https://www.blocked.org.uk/control/ispreports/BT-Strict/http://youtube.com/user/PewDiePie
https://www.blocked.org.uk/control/ispreports/Sky/http://www.wrpape.co.uk
https://www.blocked.org.uk/control/ispreports/Vodafone/http://www.wrpape.co.uk
https://www.blocked.org.uk/control/ispreports/VirginMedia/http://kodiemovies.com
https://www.blocked.org.uk/control/ispreports/Vodafone/http://kodiemovies.com
https://www.blocked.org.uk/control/ispreports/BT-Strict/http://www.evans.co.uk
https://www.blocked.org.uk/control/ispreports/Sky/http://www.videorolls.com
https://www.blocked.org.uk/control/ispreports/BT-Strict/http://toxicwap.com
https://www.blocked.org.uk/control/ispreports/BT/http://kodiemovies.com
https://www.blocked.org.uk/control/ispreports/EE/http://kodiemovies.com
https://www.blocked.org.uk/control/ispreports/O2/http://kodiemovies.com
https://www.blocked.org.uk/control/ispreports/Sky/http://kodiemovies.com
https://www.blocked.org.uk/control/ispreports/TalkTalk/http://kodiemovies.com
https://www.blocked.org.uk/control/ispreports/Three/http://kodiemovies.com

**Comment by JimKillock** [comment]
@Mike-ORG I'm not sure about the reviewer, but we can close these by using the 'cancel and resend' feature. i will go through some of these now.

**Comment by JimKillock** [comment]
I think I have dealt with all of those now. @dantheta on cancelling reports, it seems all reports are cancelled. In some cases this was not necessary. Is it possible to change this to cancel a specific ISP report, or just those which are unresolved?

**Comment by dantheta** [comment]
Yep, that should be doable.  I've restored the previous statuses for the sent/auto-closed/unblocked/no-decision reports from the list above.

**Comment by dantheta** [comment]
Spun out new issue for more specific cancellation.
